app/serializers/cache/heading_serializer.rb

```diff
@@ -143,7 +158,7 @@ def as_json
               measure_type_id: measure.measure_type.measure_type_id,
               description: measure.measure_type.description,
             },
-          }
+          }.merge(additional_code)
```

Comment:
It'd be cleaner to just add this to the body of the Hash as far as I can make out

Reply:
@willfish This was actually my suggestion - its not possible to add it into the hash because the the contents of that `additional_code` hash are conditional? Sometimes they are an empty hash, sometimes there are keys -> if it was one level down you could just include it in the hash but unfortunately thats not the case here?